homeassistant/components/wallbox/helpers.py

```diff
@@ -0,0 +1,59 @@
+"""Helpers for Wallbox integration."""
```

Comment:
In general, not a fan of a seperate helper file, I think the functions can be reduced in size and moved to the coordinator

Reply:
Actually I had it on the coordinator first, but decided to make a helper (I'm more fan of smaller files).
However, you're the code owner, so I will do the change 👍